qt/widgets/mplcpp/test/PlotTest.h

```diff
@@ -0,0 +1,157 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
A great set of tests. These are important for all options because the kwargs names are tied to the names in `mantidqt.plotting.functions` and if they change there we need something to break. I think this covers everything at the moment. :+1: